### DISCUSSION TITLE
v1.8.7 - Bug Fixes

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#909] [General]
* Fixed audio break up with some apps on lower sample rates
* Fixed app asking to install driver after every restart
* Fixed Driver not installing on some Macs

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/v1.8.7'>v1.8.7 - Bug Fixes</a>.</em>

**Comment by angusgastle** [comment]
Getting better, but not there yet. Still flipping back and forth before it errors out.

**Comment by angusgastle** [comment]
At least now I can exit without having to restart.
